### PR TITLE
slings now pack a punch when ascended

### PR DESCRIPTION
**Pull request by ynot01**
### Intent of your Pull Request

60 -> 160

#### Changelog

:cl:  
tweak: Shadowling ascendant melee damage increased
/:cl:

### CODE REVIEW

**Comment by monster860** [comment]
thats a lot of damage

**Comment by Hopekz** [comment]
one-punch-man-season-2-episode-6

**Comment by Bibby0110** [comment]
Why

**Comment by alexkar598** [comment]
Make it less tho, like 140, it's more fun to two shot people than one shot

**Comment by Hopekz** [comment]
If we're keeping it at 200 change it to something like 2000 since it 1shots anyway might as well demolish someone unless you're doing Alex's 2 shot

**Comment by Bibby0110** [comment]
smh, they already have abilities for AOE lightning, instant gib, instant enthrall

**Comment by Hopekz** [comment]
> smh, they already have abilities for AOE lightning, instant gib, instant enthrall

+1 to this
Hitting someone twice is now actually an inconvenience compared to literally making them instantly explode.

Not sure why we're going with 2 punches but its not a "bad" thing IMO just didn't make too much sense. 

Maybe it's for the ascended that like to play around with their prey? The unchanged punch did this well already.

**Comment by Darkstick** [comment]
I'd say remove their ability to instant gib and make punching gib instead. Punching someone that hard sounds more satisfying to me than just staring at them so hard they implode.

**Comment by ShadowDeath6** [comment]
I prefer it as it is now since they get crippled by the slow and have to hobble around at like 30% movespeed while I laugh at them

File: yogstation/code/modules/antagonists/shadowling/ascendant_shadowling.dm [hunk 1]
```diff
@@ -18,8 +18,8 @@
 	response_disarm = "flails at"
 	response_harm   = "flails at"
 	harm_intent_damage = 0
-	melee_damage_lower = 60 //Was 35, buffed
-	melee_damage_upper = 60
+	melee_damage_lower = 160 //Was 35, buffed
```
Comment (Idinuum) [inline yogstation/code/modules/antagonists/shadowling/ascendant_shadowling.dm]:
```suggestion
	melee_damage_lower = 160 //Was 60, buffed
```

Comment (nesisknesis) [inline yogstation/code/modules/antagonists/shadowling/ascendant_shadowling.dm]:
```suggestion
	melee_damage_lower = 160 //Was 60, buffed
```

**Review by nesisknesis — CHANGES_REQUESTED** [review]

**Pull request merged**